lib/src/roulette_group.dart

```diff
@@ -45,6 +45,26 @@ class RouletteGroup {
     return RouletteGroup(units);
   }
 
+  /// Helper function to create a even [RouletteGroup].
+  /// [itemCount] is the number of items in the group.
+  /// [iconBuilder] is a function that return the icon of the unit.
+  /// [colorBuilder] is a function that return the color of the unit.
+  factory RouletteGroup.uniformIcons(
+    int itemCount, {
+    IndexBuilder<IconData>? iconBuilder,
+    IndexBuilder<Color>? colorBuilder,
```

Comment:
Maybe we also need a `iconColorBuilder` here to control the icon color? 🤔️

Reply:
That would be a good idea, but is there any textColorBuilder for texts too ?

Reply:
There is a `textStyleBuilder` to support it.

Reply:
Added `textStyleBuilder`